### ISSUE TITLE
Translation Extractor not working in Symfony 4

### ISSUE THREAD

**Issue by fa-dd** [symfony/symfony#25062]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | yes
| Symfony version  | 4.0.0-BETA4

The CLI command translation:update is looking for the "/Resources/" folder which does no longer contain the templates as they are now moved to "templates" folder.
This might also affect storage location for translation files which are now in "translations".

Reproduction:
- create a template containing translatable string in templates folder
- run CLI command translation:update
- message will indicate that no translation messages were found

**Comment by nicolas-grekas** [comment]
ping @yceruto, as you worked on this part recently?

**Comment by chalasr** [comment]
Related to https://github.com/symfony/symfony/issues/24807 and https://github.com/symfony/symfony/pull/24590

**Comment by yceruto** [comment]
Yeap, the related trans commands aren't compatible with bundle-less apps and new paths. There is a PR in progress.

**Comment by yceruto** [comment]
@fa-dd could you try, please, this PR https://github.com/symfony/symfony/pull/25065 and check out if both translation commands work fine in your project?

**Comment by fa-dd** [comment]
I tried PR #25065 but in Symfony 4 it is still the same (not working) as this PR is still looking into "Resources" folder.

I got it working very early this morning by including a check for `KERNEL::MAJOR_VERSION >= 4` when setting the paths to the base folders for templates and translations (leaving those of bundles untouched). Don't know if there's a possibility to check whether the Flex directory structure is used, but as it is used >=4 I chose to check for that.
